### DISCUSSION TITLE
关于新版通知栏强制常驻

### DISCUSSION THREAD

**Discussion by xgd1062** [LSPosed/LSPosed#2342] [Q&A] [answered]
昨天更新了最新版1.8.6，打开之后通知栏强制常驻一个通知还无法取消，试问此举是何意思？

**Comment by JasonKhew96** [comment, marked as answer]
> 新的管理器打开方式
有部分设备因内核问题无法创建快捷方式，现在 LSPosed 切换到从通知打开管理器。此通知一直存在，也**可以在管理器设置中关闭通知**。
https://github.com/orgs/LSPosed/discussions/2337

**Reply by kimi-1101** [reply]
好的，但是关闭状态通知必须创建一个快捷方式，起码我觉得关闭一个常驻通知比多一个快捷方式更重要。我使用抽屉式的桌面是为了简洁，所以多一个图标还是有点心里痒痒的。
